test/messaging/components/FolderNav.unit.spec.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import FolderNav from '../../../src/js/messaging/components/FolderNav.jsx';
+
+const props = {
+  currentFolderId: 0,
+  folders: [
+    {
+      count: 3,
+      folderId: 0,
+      name: 'Inbox',
+      systemFolder: true,
+      unreadCount: 0
+    },
+    {
+      count: 3,
+      folderId: -2,
+      name: 'Drafts',
+      systemFolder: true,
+      unreadCount: 3
+    },
```

Comment:
Should probably add some personal folders to test that it correctly organizes them into the 'My folders' sublist.